corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml

```diff
@@ -154,10 +154,11 @@
   description: Version of mobile UCRs to use. Read more on the  <a target="_blank" href="https://help.commcarehq.org/display/saas/Moving+to+Mobile+UCR+V2">Help Site</a>
   toggle: MOBILE_UCR
   values:
-    - ['1.0', '1.0']
-    - ['1.5', '1.5']
     - ['2.0', '2.0']
-  default: '1.0'
+  default: '2.0'
+  disabled: true
+  disabled_txt: Oops! This setting has been discontinued. We recommend you change this setting to Version 2. Upon doing so this setting will no longer appear on this page.
```

Comment:
nit: do we need "Oops"? Isn't that in case of errors?

Reply:
Good point.  Addressed in https://github.com/dimagi/commcare-hq/commit/bc58280f7311c1036094ea7b9a3b19994ef5c46e  (merged with earlier commit)